tests/postController.ts

```diff
@@ -66,4 +72,46 @@ describe('Unit testing of post controllers', function() {
 		expect(resSpy.calledOnce).to.equal(true);
 		expect(resSpy.args[0][0]).to.eql({ result: 5 });
 	});
+
+	it('should downvote post', async function() {
+		const req = {
+			cookies: {
+				refreshToken: 'testToken'
+			},
+			params: {
+				id: 'testId'
+			}
+		};
```

Comment:
Here I suggest to add the nested `describe` section to encapsulate the work with votes. And here you could create `req` just once.